vmngclient/api/templates/payloads/cisco_vpn/feature/cisco_vpn.json.j2

```diff
@@ -0,0 +1,224 @@
+{
+  "templateName": "{{ name }}",
+  "templateDescription": "{{ description }}",
+  "templateType": "cisco_vpn",
+  "deviceType": [
+    "vedge-C8000V"
```

Comment:
should device type be specified like that? isn't that a variable?

Reply:
it will be pushed in separated PR.